### ISSUE TITLE
removedPollenGrainsQuantitySingleVisit

### ISSUE THREAD

**Issue by zedomel** [BioComp-USP/rebipp-data-standard#23]
| Field | Value |
| ----- | ----- |
| **GUID** |   |
| **Rebipp Class** | rebipp:FlowerTrait |
| **Label** | removedPollenGrainsQuantitySingleVisit |
| **Definition** |  The number or enumeration value for the quantity of pollen grains removed from the anther(s) of a flower exposed to a single visitor. |
| **Comments** |  |
| **Examples** |  `243`, `550` |
| **Controlled vocabulary** |  |
| **Darwin Core Class** | Occurrence |
| **Cardinality** | One to many |
| **Reference Protocol** |  |

**Comment by fonturbel** [comment]
Is it the same as "removedPollenGrainsQuantity"??

**Comment by carmensspires** [comment]
@zedomel Is not the same thing. Here, it refers to a single visit by a floral visitor. In the other case ("removedPollenGrainsQuantity"), we refer to multiple visits.

**Comment by zedomel** [comment]
@carmensspires I think you answered the question of @fonturbel, not mine hehehe :-D

But, I have some comments here too. I'm wondering if we can remove the `single visit` state here and merge this term with `removedPollenGrainsQuantity` (#36). And following the @fonturbel  comments on #36:
>Are we able to measure this in the field? how many studies that this information?

Can anyone provide examples of datasets that have this information? I feel that this term and the related (`removedPollenGrainsQuantity`) are too specific, and I can't see (without examples/datasets) the relevance of this term in a broader sense. 

In other words, is this term frequently used to be added to the standard? Or it will be collected/used by few studies?

**Comment by zedomel** [comment]
Following what we discussed in #36  the new definition should be:
`The total number of pollen grains removed from the anther(s) of a flower exposed to a single visitor`
